tests/test_tqdm.py

```diff
@@ -37,3 +37,15 @@ def test_progress_bar():
     pbar = audeer.progress_bar([0.1])
     for step in pbar:
         time.sleep(step)
+
+
+def test_progress_bar_update():
+    r"""Ensure progress bar is refreshed.
+
+    If the progress bar has to wait for a long time
+    until it would get updated,
+    we enforce an update by a given time.
+
+    """
+    for _ in audeer.progress_bar(range(2), maximum_refresh_time=0.01):
+        time.sleep(0.05)
```

Comment:
I do not test the display output of the progress bar here, but only ensure the code is executed in the tests.

We decided against testing the expected output of the progress bar when we introduced it, as it was not straightforward to achieve and `tqdm` is a well established library. What we do instead is manual tests of the expected behavior.

Reply:
Understood. you keep `maximum_refresh_time`at a small value to override the (longer) sleep interval such you can be sure that the refresh thread callback in `def refresh` is covered by the test. 

This is ok with me. I am resolving this.